Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipService.cpp

```diff
@@ -67,30 +66,18 @@ namespace AzToolsFramework
     void PrefabEditorEntityOwnershipService::Destroy()
     {
         StopPlayInEditor();
+        Reset();
+
         m_editorSliceOwnershipService.BusDisconnect();
         m_sliceOwnershipService.BusDisconnect();
-
-        if (m_rootInstance != nullptr)
-        {
-            m_rootInstance.reset();
-            m_prefabSystemComponent->RemoveAllTemplates();
-        }
     }
 
     void PrefabEditorEntityOwnershipService::Reset()
     {
         if (m_rootInstance)
         {
-            AzToolsFramework::ToolsApplicationRequestBus::Broadcast(
-                &AzToolsFramework::ToolsApplicationRequestBus::Events::ClearDirtyEntities);
-            Prefab::TemplateId templateId = m_rootInstance->GetTemplateId();
-            m_rootInstance->Reset();
-            if (templateId != Prefab::InvalidTemplateId)
-            {
-                m_rootInstance->SetTemplateId(Prefab::InvalidTemplateId);
-                m_prefabSystemComponent->RemoveAllTemplates();
-            }
-            m_rootInstance->SetContainerEntityName("Level");
+            m_prefabSystemComponent->RemoveAllTemplates();
+            m_rootInstance.reset();
```

Comment:
This changes what Reset() would mean. Previously, it meant that we clear everything but still keep a placeholder instance. Now it means that we will be destroying the instance completely. I don't think there's anything wrong with that. In fact, it makes the code simpler and gets rid of dependency on 'ClearDirtyEntities'. Good job on that. But I recommend getting some testing done around level workflows to make sure that nothing gets broken with this change.